librocksdb_sys/build.sh

```diff
@@ -3,10 +3,14 @@
 set -e
 
 con=1
-if [[ -f /proc/cpuinfo ]]; then
-    con=`grep -c processor /proc/cpuinfo`
+if [ "$MAKE_PARALLELISM" ]; then
```

Comment:
hi @arthurprs 

Where to use MAKE_PARALLELISM here? I suggest using another PR for this approach :-)